### PR TITLE
Fix flake8 errors in user_scripts/

### LINKED ISSUE #8: Clean up style issues
Need to fix the style issues flake8 is showing.

### PR DESCRIPTION
**Pull request by madhulika95b**
@ldko @somexpert 
This MR fixes flake8 errors closes #8

### CODE REVIEW

**Review by ldko — COMMENTED** [review]
I have one minor comment for a possible change, otherwise, these style changes look good to me. :)

File: user_scripts/fielded_batch_ingest.py [hunk 1]
```diff
@@ -250,9 +266,9 @@ def surtize(orig_url, preserveCase=False):
     # 5: other host
     # 6: :port
     # 7: path
-    # group def.     1          2                           3 
-    URI_SPLITTER = "^(\w+://)(?:([-\w\.!~\*'\(\)%;:&=+$,]+?)(@))?" + \
-        "(?:((?:\d{1,3}\.){3}\d{1,3})|(\S+?))(:\d+)?(/\S*)?$"
+    # group def.     1          2                           3
+    URI_SPLITTER = r"^(\w+://)(?:([-\w\.!~\*'\(\)%;:&=+$,]+?)(@))?" + \
+                   r"(?:((?:\d{1,3}\.){3}\d{1,3})|(\S+?))(:\d+)?(/\S*)?$"
     #       4                         5      6      7
```
[the remark this answers is not in the document]
Comment (madhulika95b) [reply user_scripts/fielded_batch_ingest.py:272]: Yeah...I'll add them.

File: user_scripts/fielded_batch_ingest.py [hunk 2]
```diff
@@ -24,16 +28,17 @@ def usage():
     -h, --help, display help
     -v, --verify, verify url is valid and available""")
 
+
 def url_ingest(file_name, nominator_id, project_slug, verify_url):
     """
         Get all the urls from the file and add them as a table entry to the
-        URL table (not repeating if there is a prior entry with a surt
+        URL table (not repeating if there is a prior entry with a surt.
```
Comment (somexpert) [inline user_scripts/fielded_batch_ingest.py:35]: We should add the closing parenthesis.

File: user_scripts/fielded_batch_ingest.py [hunk 3]
```diff
@@ -79,85 +84,90 @@ def csv_ingest(file_name, nominator_id, project_slug, verify_url):
...
         yield dict((key, value) for key, value in row.items())
 
+
 def pydict_ingest(file_name, nominator_id, project_slug, verify_url):
-    """Ingests pickled dictionary into nomination tool.
+    """Ingest pickled dictionary into nomination tool.
 
-    Useful for multivalue attributes that can be stored in the dictionary 
+    Useful for multivalue attributes that can be stored in the dictionary
     as lists.
```
Comment (somexpert) [inline user_scripts/fielded_batch_ingest.py:125]: Think we should remove this blank line.

File: user_scripts/fielded_batch_ingest.py [hunk 4]
```diff
@@ -281,100 +297,105 @@ def surtize(orig_url, preserveCase=False):
...
+                req = Request(url, None, headers)
+                urlopen(req)
+            except (URLError, HTTPError):
                 print(str(e) + '; Response: ' + str(e.code) + '; skipping URL ' + url)
                 return False
         else:
             print(str(e) + '; Response: ' + str(e.code) + '; skipping URL ' + url)
             return False
-    except:  # urllib2.URLError, httplib.InvalidURL, etc.
+    except (URLError, HTTPError):  # urllib2.URLError, httplib.InvalidURL, etc.
```
Comment (somexpert) [inline user_scripts/fielded_batch_ingest.py:365]: Think we can just remove the comment.

**Review by somexpert — APPROVED** [review]
Looks good to me! :thumbsup:

**Review by ldko — APPROVED** [review]
Thank you, Madhulika!

**Pull request merged**